### PR TITLE
Adding initial implementation of Sentry integration for edX

### PR DESCRIPTION
**Pull request by blarghmatey**
- Adds an app entrypoint for lms and cms
- Adds a settings module that sets up the Sentry integration with sane defaults

### CODE REVIEW

**Review by rhysyngsun — APPROVED** [review]
generally looks good to me

File: src/ol_openedx_sentry/BUILD [hunk 1]
```diff
@@ -0,0 +1,13 @@
+python_distribution(
+    name="sentry",
```
Comment (shaidar) [inline src/ol_openedx_sentry/BUILD:2]: Is this right?

File: src/ol_openedx_sentry/app.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+
+class EdxSentryCMS(AppConfig):
+    name = "ol_openedx_sentry"
+    verbose_name = "Sentry integration for Open edX"
+    plugin_app = {
+        "settings_config": {
+            "cms.djangoapp": sentry_settings_config,
+        }
```
Comment (shaidar) [inline src/ol_openedx_sentry/app.py:29]: Why the difference in the plugin_app between LMS and CMS?

**Comment by shaidar** [comment]
👍

**Pull request merged**